src/views/nino/enter-your-number.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "nino/layout.njk" %}
+{% set backLink = "/nino/enter-your-number" %}
+{% set pageTitleName = "nino.pages.enterYourNumber.title" | translate %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% block content %}
+  <h1 class="govuk-heading-l">{{ pageTitleName }}</h1>
+  <p class="govuk-body">{{ "nino.pages.enterYourNumber.description" | translate }}</p>
+  <form action="/nino/enter-your-number" method="post">
+    {{ govukInput({
+      id: "ni-number",
+      name: "ni-number"
+    }) }}
+
+    {{ govukButton({
+      text: 'general.continue' | translate
+    }) }}
+  </form>
+{% endblock %}
```

Comment:
No worires for this PR, might be work adjusting VS Code to get you a newline at end of file on save at some point?

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Huh, I turned that off because it looked like it was conflicting with Prettier's 'format on save' and I hoped Prettier would do newlines for me. I'll have another go